docs/components/tools.md

```diff
@@ -813,7 +813,7 @@ Flags:
       --rewrite.to-relabel-config-file=<file-path>
                                 Path to YAML file that contains relabel configs
                                 that will be applied to blocks
-      --tmp.dir="/tmp/thanos-rewrite"
+      --tmp.dir="/var/folders/0b/r42s6hv96hs2w84ycr2yf7qh0000gn/T/thanos-rewrite"
```

Comment:
Seems like this causes docs CI to fail

Reply:
Thanks, looks like this happens on my machine regularly.

Reply:
Yeah I think it happens due to os.MkdirTemp, which behaves differently on mac